vyper/ast/parse.py

```diff
@@ -341,6 +341,25 @@ def visit_Expr(self, node):
 
         return node
 
+    def visit_Call(self, node):
+        """
+        Convert structs declared as `Dict` node for vyper < 0.4.0 to kwargs.
+        """
+        if len(node.args) == 1 and isinstance(node.args[0], python_ast.Dict):
+            dict_ = node.args[0]
+            kw_list = []
+
+            for key, value in zip(dict_.keys, dict_.values):
```

Comment:
even though we are guaranteed by the parser that `.keys` and `.values` have the same length, i'd like to have a sanity check that `assert len(dict_.keys) == len(dict_.values)`

Reply:
done!